mitodl/ocw_oer_export
# Update export format with feedback improvements

### What are the relevant tickets?
Part of: https://github.com/mitodl/ocw_oer_export/issues/100

Works on: 
1. CR_PROVIDER can be omitted. It's provided by the input workflow.
2. CR_PROVIDER_SET should be formatted "MIT:MIT OpenCourseWare"
3. CR_SUBJECT: map OCW's "Physical Education and Recreation" to "Physical Education" in
4. CR_SUBLEVEL "Career/Technical Education" should be replaced with "Career / Technical" (remove "Education")

### How can this be tested?
```
docker compose build
docker compose run --rm app
```

Check the output file: `private/output/ocw_oer_export.csv` and compare.
